tests/nightwatch/login.js

```diff
@@ -0,0 +1,17 @@
+describe('Login into nextcloud end-to-end test', function() {
+	before(browser => browser.navigateTo(browser.launchUrl))
+
+	it('arrives on the home page and submits credentials with login form', function(browser) {
```

Comment:
Can you replace `arrives on the home page and submits credentials with login form` by `login as admin`, please ?